src/planscape/forsys/views.py

```diff
@@ -271,6 +290,73 @@ def generate_project_areas_for_a_single_scenario(
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
 
 
+@csrf_exempt
+def generate_project_areas_prototype(
+        request: HttpRequest) -> HttpResponse:
+    try:
+        body = json.loads(request.body)
+        if not 'stand' in body or not 'project' in body:
+            raise ValueError(
+                "Either the forsys stand or project dictionary is missing")
+        forsys_output_json = {
+            'stand': body['stand'],
+            'project': body['project']
+        }
+
+        user_id = body.get('user_id', None)
+        if not (isinstance(user_id, int)):
+            raise ValueError("Must specify user_id as an integer")
+
+        scenario_id = body.get('scenario_id', None)
+        if not (isinstance(scenario_id, int)):
+            raise ValueError("Must specify scenario_id as an integer")
+
+        weighted_priorities = {
```

Comment:
Can you please add a TODO clarifying why these weighted_priorities are defined inline, and where they should come from in the future?